packages/components/pie-textarea/src/index.ts

```diff
@@ -13,8 +16,21 @@ const componentSelector = 'pie-textarea';
  * @tagname pie-textarea
  */
 export class PieTextarea extends RtlMixin(LitElement) implements TextareaProps {
+    static shadowRootOptions = { ...LitElement.shadowRootOptions, delegatesFocus: true };
+
+    @property({ type: Boolean, reflect: true })
+    public disabled?: TextareaProps['disabled'];
+
     render () {
-        return html`<h1 data-test-id="pie-textarea">Hello world!</h1>`;
+        const {
+            disabled,
+        } = this;
+
+        return html`
+            <textarea
+                data-test-id="pie-textarea"
+                ?disabled=${live(disabled)}
```

Comment:
Is `live` intended here are there updates outside of lit that can't pick this change up out of interest?